File: Implementação/Projetos/api-informacoes-cidadao/src/main/java/com/poctcc/api/model/Pessoa.java
package com.poctcc.api.model;

import java.io.Serializable;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@EqualsAndHashCode(onlyExplicitlyIncluded = true)

public class Pessoa implements Serializable {

	private static final long serialVersionUID = 1L;

	private long id;

	private String telefone;

	private Endereco endereco;

	private String email;

}

File: Implementação/Front-End/portal-cidadao/src/app/models/Pessoa.ts
import { Endereco } from "./Endereco";
import { Imovel } from "./Imovel";

export class Pessoa {
    public id: number;

	public telefone: string;

	public endereco: Endereco;

	public email: string;

	public imoveis: Imovel[] = [];
}